android-sdk/src/main/java/com/checkout/android_sdk/logging/FramesLoggingEventDataProvider.kt

```diff
@@ -1,10 +1,21 @@
 package com.checkout.android_sdk.logging
-
-import com.checkout.android_sdk.Utils.Environment
 import com.checkout.eventlogger.domain.model.MonitoringLevel
+import com.checkout.android_sdk.Utils.Environment
+import java.util.*
```

Comment:
is this even used?

Reply:
Import of java.util is for a Locale instance to pass the language details in events.

Reply:
can we just have `import java.util.Locale` instead?

Reply:
It's by default common import for all java utils classes. We can add custom imports for a specific class. However, when we add any more values to events which have dependencies on this utils class, we don't require to import it manually every time.

Reply:
Yes you're right, however this reduces visibility on which classes are being imported and used. There is also the possibility of collision in class names if imported with a wildcard.

As far as best practices go for Java/Kotlin, my understanding is that you should default to importing specific classes until you reach a certain amount from that package, eg. 4, so that the wildcard will save you lines of code.

See [this stackoverflow article](https://stackoverflow.com/questions/187453/import-package-vs-import-package-specifictype) for a discussion on it.